book/09-git-and-other-scms/sections/import-hg.asc

```diff
@@ -37,19 +37,18 @@ Bob Jones <bob@company.com>
 Joe Smith <joe@company.com>
 ----
 
-In this example, the same person (Bob) has created changesets under four different names, one of which actually looks correct, and one of which would be completely invalid for a Git commit.
-Hg-fast-export lets us fix this by adding `={new name and email address}` at the end of every line we want to change, and removing the lines for any usernames that we want to leave alone.
-If all the usernames look fine, we won't need this file at all.
-In this example, we want our file to look like this:
+In this example, the same person (Bob) has created changesets under four different names, one of which actually looks correct, and one of which would be completely invalid for a Git commit. Hg-fast-export lets us fix this by turning each line into a rule: `"<input>"="<output>"`, mapping an `<input>` to an `<output>`. Inside the `<input>` and `<output>` strings, all escape sequences understood by the python `string_escape` encoding are supported. If the author mapping file does not contain a matching `<input>`, that author will be sent on to Git unmodified. If all the usernames look fine, we won't need this file at all. In this example, we want our file to look like this:
 
```

Comment:
Please maintain the formating of one sentence per line.

Reply:
Fixed and revised patch series pushed.